gnome-extra/gnome-software/gnome-software-43.0.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson virtualx xdg
+
+DESCRIPTION="Gnome install & update software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Software https://gitlab.gnome.org/GNOME/gnome-software"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flatpak +firmware gnome gtk-doc sysprof udev +webapps hardcoded-foss-webapps hardcoded-proprietary-webapps"
+REQUIRED_USE="
+	hardcoded-foss-webapps? ( webapps )
+	hardcoded-proprietary-webapps? ( webapps )
+"
+RESTRICT="test" # See TODO below, TODO needs glib-testing-0, which I can not find.
```

Comment:
Strange. Maybe it's https://gitlab.gnome.org/pwithnall/libglib-testing ?

Reply:
Will try, thanks. I feel like I looked though half of the whole internet yesterday and couldn't find it